bin/main.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+require_relative 'game'
+puts "Hello welcome to the Tic Tac Toe Game"
+
+game = Game.new
+
+puts "What is your name? (player X)"
+game.user[:x] = gets.chomp
+puts "What is your name? (player O)"
+game.user[:o] = gets.chomp
```

Comment:
Whenever you find yourself repeating code, know that there's always a way to do it without repeating the code. Keep it DRY.

An example is having a method that asks for names. :thinking: E.g.
```ruby
def get_name(player_name)
  puts  "What is your name? #{player_name}"
  name = gets.chomp
  name = player_name if name.length < 2
 name
end
```
This could be called directly to the `game.user[:x]` and `game.user[:o]` property like:
```ruby
game.user[:x] = get_name("player X")
game.user[:o] = get_name("player O")
```
The advantage of this is that we could change how we get the player's name anytime, like for example there's a validation that sets a player name to a default name if the player inputs a name less than 2 characters. Also check the illutration below on the current effect of the game.

This is what happens when I enter no name. :point_down: 
```sh
Hello welcome to the Tic Tac Toe Game
What is your name? (player X)

What is your name? (player O)

--------------------------------------
This our board
 1 | 2 | 3 
---|---|---
 4 | 5 | 6 
---|---|---
 7 | 8 | 9 
 pick a number to set the turn 'X' 
2
--------------------------------------
Next turn
--------------------------------------
   | X |   
---|---|---
   |   |   
---|---|---
   |   |   
 pick a number to set the turn 'O'
```

Let's compare with this when a valid name is entered. :point_down: 

```sh
Hello welcome to the Tic Tac Toe Game
What is your name? (player X)
Bola
What is your name? (player O)
Ahmed
--------------------------------------
This our board
 1 | 2 | 3 
---|---|---
 4 | 5 | 6 
---|---|---
 7 | 8 | 9 
Bola pick a number to set the turn 'X' 
4
--------------------------------------
Next turn
--------------------------------------
   |   |   
---|---|---
 X |   |   
---|---|---
   |   |   
Ahmed pick a number to set the turn 'O' 
```

> Read about [DRY rule](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and use it.